artifactory/commands/generic/generic.go

```diff
@@ -93,3 +94,12 @@ func (gc *GenericCommand) SetDetailedSummary(detailedSummary bool) *GenericComma
 	gc.detailedSummary = detailedSummary
 	return gc
 }
+
+func (gc *GenericCommand) GetInclude() []string {
```

Comment:
Notice that non of the getter functions in this struct include the "Get" prefix in their names. Let's remove it here.
I actually suggest naming this function to `AqlInclue`.

Reply:
Done